array-manipulation-1/src/head.js

```diff
@@ -0,0 +1,8 @@
+/* exported head */
+// make for loop (i=0; i < array.length; i++)
+// return array[0];
+
+function head(array) {
+  for (var i = 0; i < array.length; i++);
```

Comment:
this loop is doing nothing for you. It doesn't even have a code block. It is just doing nothing over and over again for each item in the array.